src/ui/public/styles/base.less

```diff
@@ -289,14 +289,102 @@ table {
 
 //== SavedObjectFinder
 saved-object-finder {
+  .row {
+    background-color: @kibanaGray6;
+    padding: 10px;
+    display: flex;
+    flex-direction: row;
+  }
+
   .form-group {
     margin-bottom: 0;
+    float: left;
+    flex: 8;
+
     input {
-      .border-bottom-radius(0);
+      border: none;
+      padding: 5px 0px;
+      border-radius: @border-radius-base;
+      text-transform: capitalize;
+    }
+
+    span {
+      padding: 5px 0px;
+      background-color: @white;
+      border: none;
+
+      i {
+        color: @kibanaGray3;
+        width: 15px;
+      }
+    }
+  }
+
+  #finder-hit-count, #finder-manage-object {
```

Comment:
Also, these styles are specific to the saved object finder, put them under `saved-object-finder`